core/src/com/mygdx/battleship/BattleshipBots/BattleshipBot.java

```diff
@@ -4,13 +4,51 @@
 import com.mygdx.battleship.MoveResult;
 
 public interface BattleshipBot {
+    
+	private String[][] shipsLocations;
+	private String move;
+	private HashTable<String,String> botShots; //bot's shots
+	private HashTable<String,String> opponentShots; //oponents shots
+	
+	
+	BattleshipBot() {
+		botShots = new HashTable<String,String>;
+		opponentShots = new HashTable<String,String>;
+		shipsPlacements();
+		nextMove();
+	}
+	
+	/**
+	 * Decide ship placement.
+	 * Sets ship placement variable.
+	 */
+	public void shipPlacements();
```

Comment:
This doesn't need to be part of the interface. If you want to have a method like this, it should be called in the bots' constructors and be private to the bots.